fuel-relayer/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+pub mod config;
+pub mod log;
```

Comment:
Seems like these eth logs aren't needed by anything else other than the relayer?

```suggestion
pub(crate) mod log;
```